### ISSUE TITLE
IPv6 addresses and prefixes not read by network-importer

### ISSUE THREAD

**Issue by smaya-netactuate** [networktocode/network-importer#305]
### Environment
* Python version:  3.8.10
* Network Importer version:  3.1.0

### Steps to Reproduce
1. Run network-importer on a device with IPv6 and IPv4 addresses assigned to interfaces
2. See that network-importer imports the IPv4 addresses but not the IPv6 addresses

### Expected Behavior
network-importer imports both IPv6 and IPv4 addresses

### Observed Behavior
IPv6 addresses are ignored. This appears to be due to the lack of IPv6 support in Batfish but is there any way to work around the issue and get network-importer to import IPv6 addresses?

**Comment by jvanderaa** [comment]
You are right that this is from the lack of IPv6 support in batfish. We will make this a strong consideration for feature enhancement as we look to build Network-Importer into a version 2.

@scetron

**Comment by scetron** [comment]
Thanks for your input @smaya-netactuate. Yes, this is a limitation of Batfish in this instance. There are ways to get around this, but it requires a lot of custom code and parsing. We will look to change this future.
